src/components/HistoryTables/ActorState/index.tsx

```diff
@@ -69,12 +69,16 @@ export const ActorState = ({ address: addressProp }: ActorStateProps) => {
   // Get actor state with descriptors
   const describedState = useMemo<DataTypeMap | null>(() => {
     try {
-      return actorData ? describeLotusActorState(actorData) : null
+      return actorName && actorData?.State
+        ? describeActorState(actorName, actorData.State)
+        : null
     } catch (e) {
-      logger.error(e)
+      logger.error(
+        `Failed to describe actor state for address ${address}: ${e.message}`
```

Comment:
Might be helpful to have the `networkName` here

Reply:
Good call